doc/sphinx/using/libraries/writing.rst

```diff
@@ -90,3 +91,13 @@ deprecated compatibility alias using :cmd:`Notation (abbreviation)`
    .. coqtop:: all warn
 
       Check (foo 0).
+
+Associating a comment to a library file
+---------------------------------------
+
+The attribute :term:`information` can be used to associate a comment
+to a library file. The comment is output when the file is loaded using
+:cmd:`Require`.
```

Comment:
This does not belong to the current PR, the syntax of the attribute should be discussed in https://github.com/coq/coq/pull/18248

Reply:
> the syntax of the attribute should be discussed in #18248

There is also a 8.19 constraint involving #18032 in play.